apk-generator/v2/app/utils/notification.py

```diff
@@ -46,7 +86,15 @@ def send_mail_via_smtp_(config, payload):
         mailer.start()
         message = Message(author=payload['from'], to=payload['to'])
         message.subject = payload['subject']
-        message.plain = strip_tags(payload['html'])
-        message.rich = payload['html']
+        message.plain = message
```

Comment:
Why are you assigning `message` to a property of `message` ? Should be the message from the payload.